packages/storybook/stories/Modal.stories.jsx

```diff
@@ -12,6 +12,7 @@ const Template = args => {
   const openModal = () => setVisible(true);
   return (
     <div style={{ height: '500px' }}>
+      <h1>Testing h1 heading</h1>
```

Comment:
This was added so @CourtneyChambers77 could make sure that two `<h1>`s on the same page didn't pose a problem when one of them was in the modal.